openweathermap/weather.html

```diff
@@ -161,6 +161,7 @@
     <p>The language of the report can be set by choosing from the drop down menu. The node supports all languages currently provided by OpenWeatherMap.</p>
     <p>The current weather report sets the following properties of <code>msg.payload</code>:</p>
     <ul>
+        <li><code>id</code> - the id given to the current weather by OpenWeatherMap.</li>
```

Comment:
I'm unaware of any ordering in that list. But I can move it to the bottom if that is what you mean.
For me the id is important because there are alternate icons for OpenWeatherMap that can be found on the Internet. They often use the id as the name of the icon.

Reply:
Yes please - or maybe after the entry for icon. Thanks